src/request/import/index.html

```diff
@@ -155,8 +155,8 @@ <h1 data-i18n="import-upload-login-file" class="nq-h1">Upload your Login File</h
                     <button data-i18n="import-file-button-words" class="nq-button-s" id="goto-words">
                         Login with Recovery Words
                     </button>
-                    <a class="nq-text-s nq-blue" id="goto-create" href="#">
-                        <span data-i18n="import-create-account">Create new account</span>
+                    <a class="skip nq-link" id="goto-create" href="#">
```

Comment:
This line should go into the skip button PR and be removed here.

Reply:
@nibhar I added this skip button to its PR (https://github.com/nimiq/keyguard-next/pull/282/commits/c25b1f30ff9994b4e61cc6fa422b746c010b6b9c), you can remove this change from this PR.